.circleci/config.yml

```diff
@@ -178,48 +178,6 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
-  e2e-pilot: #auth + v1alpha1 + v1
```

Comment:
Remove these from prow as well?

Reply:
yes, that would speed up our PR tests a lot.

Reply:
Yes, someone familiar with prow should do it - not sure who is maintaining the equivalent configs.

Reply:
We need these removed from Prow or we need to stop marking `istio-pilot-e2e` as required.